borg/testsuite/repository.py

```diff
@@ -100,13 +100,15 @@ def test_single_kind_transactions(self):
         self.repository.close()
         # replace
         self.repository = self.open()
-        self.repository.put(b'00000000000000000000000000000000', b'bar')
-        self.repository.commit()
-        self.repository.close()
+        with self.repository:
+            self.repository.put(b'00000000000000000000000000000000', b'bar')
+            self.repository.commit()
+            self.repository.close()
```

Comment:
Yeah, that's superfluous. close() is idempotent, so won't blow up either, but it doesn't belong there.